File: LeetCode/1768/1768.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        """
        :type word1: str
        :type word2: str
        :rtype: str
        """
        len1 = len(word1)
        len2 = len(word2)

        output = ""
        length = len1 if len1 > len2 else len2

        for i in range(length):
            if (i < len1):
                output += word1[i]
            if (i < len2):
                output += word2[i]
        
        return output

File: LeetCode/1768/1768.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string output = "";
        int len1 = word1.length();
        int len2 = word2.length();
        for(int i = 0; i < ((len1 > len2) ? len2 : len1); i++){
            output += word1[i];
            output += word2[i];
        }
        if(len1 < len2){
            output += word2.substr(len1);
        }else if(len1 > len2){
            output += word1.substr(len2);
        }
        return output;
    }
};
